src/gui/redux/docgen/docgen.selectors.js

```diff
@@ -1,12 +1,6 @@
 import { createSelector } from 'reselect'
-import docgen from './docgen'
+import { docgen } from '../store'
 
-const getDocgen = state => docgen.get()(state) || {}
-
-const getProps = createSelector(getDocgen, data => data.props || {})
-
-const getPropsKeys = createSelector(getProps, props => Object.keys(props))
-
-const getProp = name => createSelector(getProps, props => props[name])
-
-export { getDocgen, getProps, getPropsKeys, getProp }
+const getProps = state => docgen.get()(state).props
```

Comment:
add default value on `data.props`
```es6
const getProps = state => docgen.get()(state).props || {}
```
Because when docgen data is not fetched yet, `getPropsKeys` fails